main_controller_ip: '192.168.8.200'

robot_ip: '192.168.8.100'

# Used for ssh tunnelling
port_number: 4124
